src/main/java/io/github/bedwarsrel/BedwarsRel/Shop/Specials/ArrowBlocker.java

```diff
@@ -0,0 +1,145 @@
+package io.github.bedwarsrel.BedwarsRel.Shop.Specials;
+
+import java.util.ArrayList;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.scheduler.BukkitRunnable;
+import org.bukkit.scheduler.BukkitTask;
+
+import com.google.common.collect.ImmutableMap;
+
+import io.github.bedwarsrel.BedwarsRel.Main;
+import io.github.bedwarsrel.BedwarsRel.Game.Game;
+import io.github.bedwarsrel.BedwarsRel.Utils.ChatWriter;
+import io.github.bedwarsrel.BedwarsRel.Utils.Utils;
+
+public class ArrowBlocker extends SpecialItem {
+
+  private int livingTime = 0;
+  private Player owner = null;
+  private BukkitTask task = null;
+  private Game game = null;
+  public boolean isActive = false;
+
+  public ArrowBlocker() {
+    super();
+
+    this.game = null;
+    this.owner = null;
+  }
+
+  @Override
+  public Material getItemMaterial() {
+    return Utils.getMaterialByConfig("specials.Arrow-Blocker.item", Material.BARRIER);
+  }
+
+  public int getLivingTime() {
+    return this.livingTime;
+  }
+
+  public Game getGame() {
+    return this.game;
+  }
+
+  public Player getOwner() {
+    return this.owner;
+  }
+
+  @SuppressWarnings("deprecation")
+  public void create(Player player, Game game) {
+    this.game = game;
+    this.owner = player;
+
+    int breakTime = Main.getInstance().getIntConfig("specials.arrow-blocker.protection-time", 10);
```

Comment:
Hyphens `-` are not allowed in Java variable names.
I guess you meant `protectionTime`. :smile_cat: